test/cli.js

```diff
@@ -15,7 +15,10 @@ function execCli(args, dirname, cb) {
 		dirname = path.join(__dirname, dirname);
 	}
 
-	var env = {};
+	var env = {
+		// This probably should be set only for the corresponding test
```

Comment:
Probably by adding an `env` argument after `dirname` and before `cb`. It should be optional though, no changes to the other tests should be necessary.

Could leave `dirname` required if `env` is provided, so `execCli([], { NODE_PATH: 'foo' })` is invalid.